src/libs/snowdayCalculator.ts

```diff
@@ -17,15 +17,18 @@ const snowdays = 0;
 export async function calculate(): Promise<GetSnowdayResponse['data']> {
 	let body: string;
 	try {
-		body = await request.get({
-			url: 'http://www.snowdaycalculator.com/Apps/jsPred.php',
-			qs: {
+		const response = await axios.get('http://www.snowdaycalculator.com/Apps/jsPred.php', {
+			params: {
 				zipcode,
-				snowdays,
-				school: schoolId
+				snowdays
+				//   school: schoolId
```

Comment:
Why was this commented out?

Reply:
if you scroll a couple lines up, you see this:

```typescript
const zipcode = 63124;
// PHP back-end was throwing an error sometimes if school id was inputted
// const schoolId = 663;
const schoolId = '';
const snowdays = 0;
```

it's already noop.